### ISSUE TITLE
KorGE and Android

### ISSUE THREAD

**Issue by reymalahay** [korlibs/korge#413]
Hi all,

I am currently RTFM'ing the documentation.

I need to clarify, when the author uses `View` in this extract, which can be found [here](https://docs.korge.org/korge/reference/views/)

```It is a tree structure where View has a parent, and Container has other View as children.```

is `View` interchangeable with `View` in `Android` speak?  For example, can an `Android` `Fragment` contain a `KorGE` scene? 

Please advise.

Thanks in advance,
rey malahay

**Comment by soywiz** [comment]
No, nothing like that. They are completely different. KorGE Views are different since it can run on several devices not only Android

**Comment by reymalahay** [comment]
@soywiz,


thank you for the response.

lastly, before i close this issue, just one more question: is the KorGE project structure compatible with a typical Android Studio project structure?

A KorGE project structure:

├── gradle
│   └── wrapper
└── src
    └── main
        ├── java
        └── res


 A typical Android Studio project structure:

├── app
│   ├── build
│   │   ├── generated
│   │   ├── intermediates
│   │   ├── outputs
│   │   ├── reports
│   │   ├── test-results
│   │   └── tmp
│   ├── libs
│   └── src
│       ├── androidTest
│       ├── main
│       └── test
├── docs
└── gradle
    └── wrapper

The reason i ask is because how does one integrate `KorGE` into an existing Android project that was built using Android studio. Especially if the application needs to access files using `resourcesVfs`


please advise.


kind regards,
rey malahay


> No, nothing like that. They are completely different. KorGE Views are different since it can run on several devices not only Android.
> 
> For questions like this that are not bugs or feature requests, please ask them in our discord server (more active): https://discord.korge.org/ or in the forums (less active) https://forum.korge.org/



> No, nothing like that. They are completely different. KorGE Views are different since it can run on several devices not only Android.
> 
> For questions like this that are not bugs or feature requests, please ask them in our discord server (more active): https://discord.korge.org/ or in the forums (less active) https://forum.korge.org/
